bin/extract-strings.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
```

Comment:
This script exists to continue supporting the l10n flow and `npm run l10n:check` by generating just the FTL in isolation from the rest of the webpack build